README.md

```diff
@@ -1,40 +1,75 @@
+Distributed Key Generation is a cryptographic process that aims to solve the problem of coordinating `N` parties to cryptographically sign and verify signatures without relying on Trusted Third Parties.
+The process is demonstrated to be successful in successfully computing a key pair in the presence of a number `T` attackers in a decentralized network.
+To do so, this algorithm generates a public key, and a secret key of which no single party knows, but has some share of.
+The involvement of many parties requires Distributed key generation to ensure secrecy in the presence of malicious contributions to the key calculation.
 
-```json
-[
-  {
-    "id": 1,
-    "public_key": "LS0tLS1CRUdJTiBSU0....",
-    "ip": "http://localhost:3030"
-  },
-  {
-    "id": 2,
-    "public_key": "LS0tLS1CRUdJTiB....",
-    "ip": "http://localhost:3031"
-  }
-]
+For more information about DKG in general, [please visit this page](https://en.wikipedia.org/wiki/Distributed_key_generation).
+
+### DKG tool by SSV
+
+The SSV team built this tool leveraging [drand](https://drand.love/)'s DKG protocol implementation ([please visit their documentation](https://drand.love/docs/cryptography/#setup-phase) for more details on it). This implementation operates under the assumption of a p2p network, allowing operators to communicate.
+
+The `ssv-dkg` was built to lift this assumption and provide a communication layer that centered on an Initiator figure, to facilitate communication between operators. The introduced potential risk for centralization and bad actors is handled with signatures and signature verifications, as explained in the [Security notes](#security-notes) section.
+
+Finally, the outcome of the DKG ceremony is a BLS key pair to be used for validator duties by operators on the ssv.network. As such, the tool ends the process by creating a deposit file to activate the newly created validator key pair, and proceeds to generating the payload for the transaction, and triggering the transaction itself.
+
+## Quick-start
+
+### Overview
+
+In order for the DKG protocol to execute successfully:
+* all the chosen Operators must be running the `ssv-dkg` tool as Operators
+* separately, an Initiator (one of the Operators, or a separate entity), starts the DKG ceremony by running the `ssv-dkg` tool with the `init` parameter
+* the tool automatically exchange data between the interested parties, as outlined in the [Architecture](#basic-flow-description) section, until the key shares are created
+
+### Build
+
+```sh
+make install
 ```
 
 ### Operator
 
-The dkg-operator is ran by a SSV operator, an Operator RSA private key is a requirement.
-The operator is able to participate in multiple DKG ceremonies in parallel.
+SSV Operators typically play the role of dkg-operators, running the `ssv-dkg` tool as Operators. As a result, it is an Operator RSA private key is a requirement. A dkg-operator is able to participate in multiple DKG ceremonies in parallel.
```

Comment:
this is a lot of times saying operator/s in one sentence. also its not very clean and this sentence sounds funny - it is an Operator RSA private key is a requirement

Reply:
yes, there's been some delete-add text that messed up the sentence: "it is an Operator RSA private key is a requirement" is broken English 😆 
Thanks for spotting that